xml/System.Net.Mime/MediaTypeNames+Application.xml

```diff
@@ -64,8 +64,15 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Specifies that the <see cref="T:System.Net.Mime.MediaTypeNames.Application" /> data is in JSON format.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The <xref:System.Net.Mime.MediaTypeNames.Application.Json> member designates that the data is in JSON format.
+  
+ ]]></format>
+        </remarks>
```

Comment:
```suggestion
        <remarks>To be added.</remarks>
```

Reply:
Looks like duplicate text to summary - other formats like PDF, RTF, ZIP follow the same pattern